toc.md

```diff
@@ -714,30 +880,42 @@ module Workerholic
 end
 ```
 
-In our second iteration, we decided to use a hash instead. Same as before, using the beginning of the day timestamps as hash fields, and this way we push the computation logic down to the Redis level when we retrieve our data. Redis also has a very nice convenient hash method for incrementing fields.
+By doing the above we avoid concurrency issues by relying on Redis, which uses a write lock when executing the `HINCRBY` command.
+Once we have figured out how we wanted to store the data, we need to think about how many data points to store?
```

Comment:
Once we figured out how we wanted to store the data, we needed to think about how many data points to store.